internal/jujuapi/api_test.go

```diff
@@ -32,7 +32,7 @@ func (s *apiSuite) SetUpTest(c *gc.C) {
 
 	mux := http.NewServeMux()
 	mux.Handle("/api", jujuapi.APIHandler(ctx, s.JIMM, s.Params))
-	mux.Handle("/model/", jujuapi.ModelHandler(ctx, s.JIMM, s.Params))
+	mux.Handle("/model/", http.StripPrefix("/model", jujuapi.ModelHandler(ctx, s.JIMM, s.Params)))
```

Comment:
Why strip prefix?

Reply:
It's because of the change in [jujuapi/api.go](https://github.com/canonical/jimm/pull/1344/files#diff-fe36fdbf3a8c97b34cc281bca11eb3bba8ff8fe8bfc80b84e06abf48c5870391R39), before we just matched `/` onto the api proxy. But now we have some different endpoints, so we just define the relative paths in there and the at the top level here (and in service.go where we also do this), we strip the prefix before passing on the rest of the path. This [SO post](https://stackoverflow.com/questions/27945310/why-do-i-need-to-use-http-stripprefix-to-access-my-static-files) also probably explains it better than I did.

Reply:
Doesn't stripprefix just remove r.path.URL prefixes? Your handler would work all the same? Are we using r.path.URL in the handler anywhere?

Reply:
I think if we don't use stripprefix then we need to put /model/ in the handler, this just strips the parts that are common.

Reply:
Can you try without please?

Reply:
It would work but it would mean adding /model/ to the routes in `jujuapi.ModelHandler`, I did it intentionally.

Reply:
Are you sure?

Reply:
That's what I recall happening, but this was like 2 months ago. I'll try it again now and let you know.